### DISCUSSION TITLE
Google translate ready copy and past sentences key translations

### DISCUSSION THREAD

**Discussion by bitdom8** [ivanhofer/typesafe-i18n#325] [Q&A] [answered]
Having typesafe is great. Thanks. However having google translate ready sentences in translation files would be great. For instance, 

"This_is_your_password"

would hard to copy and paste translate and let translators edit or proofread. Thanks

**Comment by ivanhofer** [comment, marked as answer]
If I understand correctly you are referring to the key-based translations that are used in most examples.
e.g.
```json
{
   "password_label": "This is your password"
}
```

You can choose any key you want. So you could also write:
```json
{
   "This is your passsword": "This is your password"
}
```
for your base translation and tranlslate it to other languages e.g. german:
```json
{
   "This is your passsword": "Das ist dein Passwort"
}
```

Was this your question?

**Reply by bitdom8** [reply]
Привет, yes Ivan, much appreciated. Спасибо
